src/pages/en/about/teams.mdx

```diff
@@ -4,88 +4,99 @@ description: Learn about Quilt's team structure, and find out who's involved.
 
 ## Community Managers
 
-Community Managers are administrative Community Team members that deal with the day-to-day tasks of maintaining Quilt's community spaces, as well as supporting and managing the rest of the community team. They're responsible for properly running elections, interviewing and training new Community Team members, managing the structure of Quilt's community spaces, writing and receiving feedback on Quilt's community policies -- while also acting as a moderator and inheriting the responsibilities from that position.
+Community Managers are administrative Community Team members that deal with the day-to-day tasks of maintaining Quilt's community spaces, as well as supporting and managing the rest of the community team.
+They're responsible for properly running elections, interviewing and training new Community Team members, managing the structure of Quilt's community spaces, writing and receiving feedback on Quilt's community policies -- while also acting as a moderator and inheriting the responsibilities from that position.
 
-<div class="moderator team-grid">
-	<TeamMembers
-		list={["AppleTheGolden", "gdude2002", "MrMangoHands", "Starchild", "Leah"]}
-	/>
-</div>
+<TeamMembers
+	type="moderator"
+	list={["AppleTheGolden", "gdude2002", "MrMangoHands", "Starchild", "Leah"]}
+/>
 
 ## Moderators
 
 Quilt's Moderator team is responsible for daily moderation tasks in Quilt community spaces. This includes conflict resolution, defensive and reactive moderation tasks and general rule and policy enforcement. Moderators are the backbone of any Community Team, and Quilt's is no exception -- a diverse, active Moderator team has always been instrumental for Quilt's success.
 
-<div class="moderator team-grid">
-	<TeamMembers
-		list={[
-			"Ember",
-			"Forkk",
-			"Larry",
-			"Octal",
-			"SilverAndro",
-			"Southpaw",
-			"snoozestudios",
-			"tempest",
-			"Redblueflame",
-		]}
-	/>
-</div>
+<TeamMembers
+	type="moderator"
+	list={[
+		"Ember",
+		"Forkk",
+		"Larry",
+		"Octal",
+		"SilverAndro",
+		"Southpaw",
+		"snoozestudios",
+		"tempest",
+		"Redblueflame",
+	]}
+/>
 
 ## Events Team
 
 The Events Team is responsible for organising and managing events within our community spaces, which can include anything from AMAs to fully-blown online modding conventions. While they aren't Moderators, they are always backed up by the rest of the Community Team, and also have the ability to directly moderate events.
 
-<div class="community team-grid">
-	<TeamMembers list={["Fusion", "Lemma", "Octal", "Bubblie", "Pyrrha"]} />
-</div>
+<TeamMembers
+	type="community"
+	list={["Fusion", "Lemma", "Octal", "Bubblie", "Pyrrha"]}
+/>
+
+### Meeting Leaders
+
+Meeting Leaders are in charge of our monthly developer meetings.
+Their responsibilities include organizing, leading and guiding meetings, collecting talking points from developers and making any changes to our meetings system.
+
+<TeamMembers type="community" list={["Bubblie"]} />
```

Comment:
Gonna note that @TibiNonEst is also a meeting leader, though I presume their addition to the listing is pending their graduation from trainee status.

Reply:
> though I presume their addition to the listing is pending their graduation from trainee status.
Yeah exactly

Reply:
Noting that they've graduated from their trainee status as of today, so they should be added to the Meeting Leader listing now.